web/tests/pm_list_data.test.js

```diff
@@ -233,18 +233,18 @@ test("get_list_info_unread_messages", ({override}) => {
     check_list_info(list_info, 15, 2, [
         "Iago",
         "Cardelio",
-        "Alice, Bob, Cardelio, Zoe",
-        "Alice, Bob, Cardelio",
-        "Alice, Cardelio, Zoe",
-        "Bob, Cardelio, Zoe",
-        "Alice, Cardelio",
-        "Bob, Cardelio",
-        "Cardelio, Zoe",
-        "Alice, Bob, Zoe",
-        "Alice, Zoe",
-        "Bob, Zoe",
+        "Alice, Bob, Cardelio, and Zoe",
+        "Alice, Bob, and Cardelio",
+        "Alice, Cardelio, and Zoe",
+        "Bob, Cardelio, and Zoe",
+        "Alice and Cardelio",
+        "Bob and Cardelio",
+        "Cardelio and Zoe",
+        "Alice, Bob, and Zoe",
+        "Alice and Zoe",
+        "Bob and Zoe",
         "Zoe",
-        "Alice, Bob",
+        "Alice and Bob",
         "Bob",
     ]);
```

Comment:
Do the test changes in this commit go with the `inbox_ui.js` changes?

`pm_list_data` doesn't import that module....

Reply:
My bad, I found that it should come with the commit of people.ts, as the change is caused due the change in get_recipients function of people.ts
I'll add it with commit for people.ts.

Reply:
I found one more test ```buddy_data.test.ts``` which was in wrong commit. Updated it with ```People``` commit.